nahusenayElias/Team_4
# DRD-11: Add Mautic Instace to the repo

### Related ticket:
[DRD-11](https://edu-team4-react24k.atlassian.net/jira/software/projects/DRD/boards/1/backlog?selectedIssue=DRD-11)

### In this PR:

Added a Mautic instance to the repository

### Testing instructions:

- checkout branch
- lando start
- cd to mautic-lando directory
- lando start
- open the localhost link and the Mautic dashboard should be visible to you
![image](https://github.com/user-attachments/assets/d903660b-e824-403d-b6d8-25eca90da4be)
